sample.env

```diff
@@ -0,0 +1 @@
+DATABASE="database name"
```

Comment:
Please leave a new line at the end of this file and rename it to `.env.example`. Also use [this package](https://github.com/codeshifu/sync-dotenv) written by @codeshifu to sync our `.env` file with `.env.example`

Reply:
resolved